src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebeforecasedefaultcolon/InputNoWhitespaceBeforeCaseDefaultColonEnumAndStrings.java

```diff
@@ -83,7 +83,7 @@ enum T {
     }
 
     static {
-        int x;
+        int x = 1;
```

Comment:
right decision.

in this snippet we do not care about initialization. we focused on `NoWhitespaceBefore`
```
        int x = 1;
        T e = T.B;

        boolean t8 = (switch (e) {
            case A
                : // violation
                x = 1;
                yield true;
            case B : // violation
                yield (x = 1) == 1 || true;
            default
                    :yield false; // violation
        }) && x == 1;
```